releng/freemarker-ide.target

```diff
@@ -23,14 +23,11 @@
       <unit id="org.junit" version="0.0.0"/>
       <unit id="org.eclipse.ui.workbench" version="0.0.0"/>
       <unit id="org.eclipse.ui.ide.application" version="0.0.0"/>
-      <repository location="https://download.eclipse.org/releases/2021-06"/>
-    </location>
-    <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="true" type="InstallableUnit">
-      <repository location="https://download.eclipse.org/eclipse/updates/4.20"/>
+      <repository location="http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.71.1.Final/REPO"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="true" type="InstallableUnit">
       <unit id="org.jboss.tools.tests" version="0.0.0"/>
-      <repository location="https://download.jboss.org/jbosstools/photon/snapshots/builds/jbosstools-base_master/latest/all/repo"/>
+      <repository location="http://download.jboss.org/jbosstools/oxygen/snapshots/builds/jbosstools-base_master/latest/all/repo"/>
```

Comment:
I don't what all these are (I most important  is the Maven build for now anyway), but why oxygen specifically?